src/components/InventoryV2/form/BatchDetailsForm.tsx

```diff
@@ -16,6 +16,8 @@ import { stateName } from 'src/types/Accession';
 import isEnabled from 'src/features';
 import { SavableBatch } from 'src/redux/features/batches/batchesAsyncThunks';
 import useSnackbar from 'src/utils/useSnackbar';
+import { useProjects } from 'src/hooks/useProjects';
```

Comment:
Moved `useProjects` into `src/hooks`.. if this is better off residing alongside `ProjectsDropdown` I can make that change. Picked `src/hooks` as a starting point for now.